onnxruntime/core/framework/onnxruntime_sequence_type_info.h

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 #pragma once
 
-#include "onnxruntime_c_api.h"
+#include "core/session/onnxruntime_c_api.h"
```

Comment:
Your other changes are all good. But with this line of change, I'm afraid we will need to fix our samples. https://github.com/microsoft/onnxruntime-inference-examples/tree/main/c_cxx . They use our C API and take an assumption that onnxruntime_c_api.h can be used individually without extra files(which is not true anymore) and folder structure. So that the samples looks as simple as possible. 

So, maybe we should put the C/C++ header files in a separated folder(no longer core/session) and add the new folder to the include path. 

What do you think?

Reply:
I think the current segregation of header files is good, we should keep it, and if we want to keep samples as is, we can add `/include/onnxruntime/core/session` install interface of `onnxruntime` cmake target.